metacat-metadata-mysql/src/main/java/com/netflix/metacat/metadata/mysql/MySqlTagService.java

```diff
@@ -217,6 +217,9 @@ public void renameTableTags(final QualifiedName name, final String newTableName)
         try {
             final QualifiedName newName = QualifiedName.ofTable(name.getCatalogName(), name.getDatabaseName(),
                 newTableName);
+            jdbcTemplate.update(SQL_UPDATE_TAG_ITEM,
+                new String[]{MysqlUserMetadataService.DELETE_MARKER + newName.toString(), newName.toString()},
+                new int[]{Types.VARCHAR, Types.VARCHAR});
             jdbcTemplate.update(SQL_UPDATE_TAG_ITEM, new String[]{newName.toString(), name.toString()},
```

Comment:
Is this a new way to soft delete? :)

Reply:
No. The issue is that when a table ABC is dropped, defintion_metadata for the table is not deleted. When we rename a table XYZ to ABC, the operation fails because there is an existing entry in defintion_metadata.
Now to fix this, I can drop the existing entry and then rename the table. Instead I rename the entry. We should revisit when we implement soft-delete.

Reply:
that's what I thought. Will our cleanup process pick them up later?

Reply:
yes.